contracts/Vault.sol

```diff
@@ -67,9 +68,10 @@ contract Vault is IVault, Initializable {
       if (_balanceOf < _emission) {
         revert InsufficientFund();
       }
-      uint256 _veEmission = (_emission * veRate) / 10000;
-
-      rewardsDistributor.distributeRewards{value: _veEmission}();
+      uint256 _veEmission = (_emission * shareRate) / 10000;
+      if (_veEmission == 0) revert ZeroEmission();
```

Comment:
if emission is 0, skip reward distribution call.

Reply:
ok